src/simulations/ModelSetup.jl

```diff
@@ -0,0 +1,18 @@
+module ModelSetup
```

Comment:
this file is only a repackaging of domains.jl and spatial_parameters.jl into a module, which eventually can do model setup

Reply:
I added a ClimaLand function which calls the ModelSetup one, so it should be OK now